server/controllers/api.py

```diff
@@ -923,6 +932,19 @@ def post(self, user, backup_id):
 
         return self.schema.store_comment(user, backup)
 
+    @marshal_with(schema.get_fields)
+    def get(self, user, backup_id):
+        backup = models.Backup.query.get(backup_id)
+        if not backup:
+            if user.is_admin:
+                restful.abort(404)
+            else:
+                restful.abort(403)
+        if not models.Backup.can(backup, user, "view"):
+            restful.abort(403)
+        return {"comments": [backup.comments]}
```

Comment:
Once you've run the ./manage.py db to seed the database that should be all you need to. 
Check out the student view - navigate over to a student submission and then view a submission - you'll get a submission like 
http://localhost:5000/cal/cs61a/su94/scheme/submissions/av29Va/

av29Va is the hashid that you can put it into your browser when you are manually testing the API. 

^  that backup won't necessarily exist in the test environment because the test environment is created separately. If you want a backup obj in the test file - checkout how `test_get_backup` does it: 
```
        self._test_backup(False)
        backup = Backup.query.first()
```